build/setup.d/20-install-packages.sh

```diff
@@ -30,4 +30,4 @@ libswitch-perl
 
 echo "Installing packages..."
 
-apt-get install -y -q --no-install-recommends -o Dpkg::Options::="--force-confold" $pkgs >>install.log
+apt-get update -y && apt-get install -y -q --no-install-recommends -o Dpkg::Options::="--force-confold" $pkgs >>install.log
```

Comment:
Could you add a comment to explain the apt options?

Reply:
I am not entirely sure myself either. I thing the options were simply inherited from the Zalando repository: https://github.com/zalando-stups/taupage/blob/master/build/setup.d/20-install-packages.sh#L45

Reply:
Ok nevermind :)